### ISSUE TITLE
Compatibility with Custom Math Pricing Extension

### ISSUE THREAD

**Issue by notjay** [organicinternet/magento-configurable-simple#238]
I've got another extension installed which allows for simple products to be priced by varying custom options and attributes.  Example, you can set the price formula to {price} + {length} x {height} x {price_per_unit}.  I've been trying to get this to work with configurable products.  does anyone have any insight on how I can take a custom option such as length and apply this to the pricing logic with this extension?

**Comment by notjay** [comment]
Additional Information:

My Pricing Logic: {length} \* {ppf} + {price}

Length is a custom option (set on the configurable product level)

PPF is Price Per foot and is an attribute set on the Associated (Simple) Product level

Price is the base price of the product (which is the Price for the Associated Product)

Any help would be greatly appreciated!

**Comment by ilinked** [comment]
I have a need to calculate both Price & Weight for customized tubular/pipe products. I would appreciate knowing what extension you are using and if you have been able to get it working with Simple Products used in Configurable Products.  

It would be ideal if the accumulative weight could also be displayed as options are selected.

Also, what modifications for SCP are available for Magento ver. 1.9.2.1

Any insight into this problem would be greatly appreciated!

**Comment by notjay** [comment]
ilinked, the extension we bought still needs customization to it.  It's Custom Math Pricing by Mico Solutions.  I wouldn't know if it would work for your situation or not but it allows to enter custom pricing logic.  There's a few others but you should also consider a developer.

**Comment by ucavus** [comment]
I don't think you need to use a maths extension for this. Magento already allows a decimal quantities, so you should specify a price per unit of length/weight and have people specify the length/weight they need when adding to cart.

**Comment by notjay** [comment]
Quantity is for the Number of products, not the length.  Using quantity for length creates a very poor customer experience and is not a solution for this issue.  I  would never order from a site that requires you to enter the length or a measurement in the quantity field.

**Comment by ucavus** [comment]
Different solutions for different problems. I'm sure you _would_ buy from a shop if you wanted an arbitrary length of a certain product, and you were able to specify exactly the length you wanted. Your description was not exactly precise, making it appear as though you want to sell arbitrary lengths. .

It sounds like you could be using standard configurable products and have attribute values add to the price, like normal.

**Comment by notjay** [comment]
My second comment sums up the pricing logic... {length} \* {ppf} + {price}.  Length is user defined via a text box.  Ppf is a fixed value.  Price is the base price.

Length x Price per Foot + Base Price

**Comment by ucavus** [comment]
Yeah, nowhere did you write your product has fixed lengths so, given the lack of clarity in your writing, allowing arbitrary decimal lengths was a valid suggestion. No one is obliged to help you, here, so drop the attitude.

**Comment by notjay** [comment]
What attitude? I was trying to clear up the confusion.

**Comment by ilinked** [comment]
Sorry I didn't get back to you sooner.  I have just gone through the
process of moving my residence & my office.  Now I can get back to the task
of focusing on my project.  It will probably take a few days to get my
thoughts realigned and again resolve this and other issues.

I will get back to you shortly with a better description and example of the
problem.

Thanks,

Bob

On Mon, Sep 14, 2015 at 8:51 PM, notjay notifications@github.com wrote:

> What attitude? I was trying to clear up the confusion.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/organicinternet/magento-configurable-simple/issues/238#issuecomment-140250531
> .

## 

Robert Philibert
(504) 717-3637
